modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc]
+
+:_content-type: REFERENCE
+[id="ztp-gitops-ztp-max-spoke-clusters_{context}"]
+= {ztp} managed cluster limits
+
+With {ztp-first}, you can manage thousands of clusters in geographically displaced regions and networks.
+The Red Hat Performance and Scale lab successfully created and managed 3500 {sno} clusters from a single {rh-rhacm-first} hub cluster in a lab environment on 4 January 2023.footnote:[link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing#single-node[Creating and managing {sno} clusters with {rh-rhacm}]]
+
+[IMPORTANT]
+====
+In real world situations, the scaling limits for the number of clusters that you can manage with a single hub cluster will vary depending on various factors affecting the hub cluster.
+====
```

Comment:
This warning looks a bit out-of-place.
Below you make it sufficiently clear that there are several factors to consider.
I don't feel there is any additional need for a big red warning sign